srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/CategoryBuilderTest.java

```diff
@@ -0,0 +1,49 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import static org.junit.Assert.*;
+import gov.va.med.srcalc.domain.model.DiscreteNumericalVariable.Category;
+import gov.va.med.srcalc.domain.model.MultiSelectOption;
+
+import org.junit.Test;
+
+public class CategoryBuilderTest
+{
+    @Test
+    public final void testBuild()
+    {
+        final CategoryBuilder builder = new CategoryBuilder();
+        
+        // Test default value.
+        assertEquals("", builder.getValue());
+        
+        // Test building with the default range.
+        final String value = "firstValue";
+        assertSame(builder, builder.setValue(value)); // note: modifying builder
+        assertEquals(
+                new Category(new MultiSelectOption(value), 100f, true),
```

Comment:
As I said before, I think the default range should be 0 to 200 as established earlier.